cmd/helmitTypes.go

```diff
@@ -21,3 +23,26 @@ type HelmitResponse struct {
 	Error    bool            `json:"error"`
 	Replicas []HelmitReplica `json:"replicas"`
 }
+
+//ShipmentStatus represents the deployed status of a shipment
+type ShipmentStatus struct {
+	Namespace string `json:"namespace"`
+	Version   string `json:"version"`
+	Status    struct {
+		Phase      string `json:"phase"`
+		Containers []struct {
+			Image    string `json:"image"`
+			Ready    bool   `json:"ready"`
+			Restarts int    `json:"restarts"`
+			State    struct {
+				Running struct {
```

Comment:
Ok, I did look at the other API, however, it didn't have the `"startedAt"` field which I thought would be useful.  It also includes logs which could potentially be a lot of data to return.